tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -431,21 +431,38 @@ private static TracerManager CreateInitializedTracer(ImmutableTracerSettings set
         private static void OneTimeSetup()
         {
             // Register callbacks to make sure we flush the traces before exiting
-            LifetimeManager.Instance.AddShutdownTask(RunShutdownTasks);
+            LifetimeManager.Instance.AddShutdownTask(new Func<Task>(RunShutdownTasksAsync));
```

Comment:
You don't need the Func right?
```suggestion
            LifetimeManager.Instance.AddShutdownTask(RunShutdownTasksAsync);
```

Reply:
done in [43e5a69](https://github.com/DataDog/dd-trace-dotnet/pull/2583/commits/43e5a694657ee3a2c735083ba0b2e8e3a33736bd)